docs/usage.md

```diff
@@ -134,14 +134,15 @@ Run `cf delete-service` to delete the service.
 ```
   $ cf delete-service my-bucket
 ```
+-- Delete policy section
```

Comment:
what is this line for?  looks like a placeholder for some other content

Reply:
Right, removed it.